src/languages/smalltalk.js

```diff
@@ -14,7 +14,7 @@ function(hljs) {
     begin: '#' + hljs.UNDERSCORE_IDENT_RE
   };
   return {
-    aliases: ['st'],
```

Comment:
This one was really important as it conflicts with ST language. I think that alias is secondary to primary language name if it match.